actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -412,36 +412,51 @@ def radio_button_tag(name, value, checked = false, options = {})
       # * <tt>confirm: 'question?'</tt> - If present the unobtrusive JavaScript
       #   drivers will provide a prompt with the question specified. If the user accepts,
       #   the form is processed normally, otherwise no action is taken.
-      # * <tt>:disable_with</tt> - Value of this parameter will be used as the value for a
+      # * <tt>:disable_with</tt> - Defaults to value attribute. Value of this parameter will be used as the value for a
```

Comment:
You haven't explained what `:disable_with` does yet, and then you say it defaults to value - how should users know what that means? :smile:

Lets put it last in the description.